packages/core/src/App/Containers/AccountSwitcher/account-switcher.jsx

```diff
@@ -248,9 +248,22 @@ const AccountSwitcher = props => {
         return getRemainingAccounts(getRealMT5());
     };
 
+    const canOpenMulti = () => {
+        if (props.available_crypto_currencies.length < 1 && !props.has_fiat) return true;
```

Comment:
Why `is_eu` check is removed here?

Reply:
Because regulated clients now can change their fiat currency.